doc/sphinx/source/guides/running-openocd-without-sudo.rst

```diff
@@ -0,0 +1,140 @@
+.. _theCore_OpenOCD_NoRoot:
+
+Running OpenOCD without root privileges
```

Comment:
The whole guide, NixOS version :smirk: 
```nix
{
  users.extraGroups.plugdev = { };
  users.extraUsers.rasen.extraGroups = [ "plugdev" "dialout" ];

  services.udev.packages = [ pkgs.openocd ];
}
```

Reply:
Could you please gimme a link or extend this message, so I can post it in this page, too? :)

Reply:
Basically...

That's how you create a group:
```nix
{ users.extraGroups.plugdev = { }; }
```

That's how you add a user to the group:
```nix
{ users.extraUsers.rasen.extraGroups = [ "plugdev" ]; }
```

And that's how you install udev rules that come with `openocd` package:
```nix
{ services.udev.packages = [ pkgs.openocd ]; }
```

Reply:
So, where to add those lines?

Reply:
NixOs configuration file (default, `/etc/nixos/configuration.nix`)

Reply:
There is only one configuration file for NixOS ;)

Reply:
Got it!